rfcs/text/0044/hash.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: hash
```

Comment:
@mjwolf we do have all hashes associated with this file stored in file.hash.*
Do you think file fits better?

Reply:
> @mjwolf we do have all hashes associated with this file stored in file.hash.* Do you think file fits better?

Yes, that would be better. @txhaflaire to do this, you'll have to add this to `hash.yml`, and then with field re-use, it will be added to the generated file schema automatically. Sorry for the confusion